plugins/storage/volume/default/src/main/java/org/apache/cloudstack/storage/datastore/lifecycle/CloudStackPrimaryDataStoreLifeCycleImpl.java

```diff
@@ -243,6 +257,10 @@ public DataStore initialize(Map<String, Object> dsInfos) {
             if (port == -1) {
                 port = 0;
             }
+            if (multi) {
+                storageHost = storageHost + (hostPath.substring(0, hostPath.lastIndexOf("/")).replaceAll("/", ","));
+                hostPath = hostPath.substring(hostPath.lastIndexOf("/")+1);
```

Comment:
```suggestion
                hostPath = hostPath.substring(hostPath.lastIndexOf("/") + 1);
```

Reply:
@GutoVeronezi   I modified the code as you requested.